apps/invoices/scripts/migrations/v1-to-v2/run-migration.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+
+import * as dotenv from "dotenv";
+import {
+  fetchCloudAplEnvs,
+  getMetadataManagerForEnv,
+  safeParse,
+  verifyRequireEnvs,
+} from "../migration-utils";
+import { ConfigV1ToV2Transformer } from "../../../src/modules/app-configuration/schema-v2/config-v1-to-v2-transformer";
+import { AppConfigV2MetadataManager } from "../../../src/modules/app-configuration/schema-v2/app-config-v2-metadata-manager";
+import { AppConfigV2 } from "../../../src/modules/app-configuration/schema-v2/app-config";
+
+dotenv.config();
+
+const runMigration = async () => {
+  verifyRequireEnvs();
+
+  const allEnvs = await fetchCloudAplEnvs();
+
+  const results = await Promise.all(
+    allEnvs.map((env) => {
+      const metadataManager = getMetadataManagerForEnv(env.saleorApiUrl, env.token);
+
+      return Promise.all([
+        metadataManager.get("app-config", env.saleorApiUrl),
+        metadataManager.get("app-config-v2"),
```

Comment:
I suggest exporting these "app-config" and "app-config-v2" tokens from somewhere.